packages/browser-client/src/Components/PeerButtons.tsx

```diff
@@ -403,7 +220,7 @@ export default function PeerButtons() {
                 width={'33%'}
                 title="Add content to offer"
                 onClick={() => {
-                  addToOffer('header')
+                  peerActions.addToOffer('header')
```

Comment:
The hover state `title` element of the "Find Header"/"Find Body" buttons doesn't match.  It says `add content to offer` which seems incorrect.